### DISCUSSION TITLE
All scores set in lazer gonna be deleted?

### DISCUSSION THREAD

**Discussion by hashelq** [ppy/osu#23143] [Q&A] [answered]
Just checked, the replay is not send when sending a score (mitmproxy).

Does this mean all scores will be deleted on release because of cheaters?

**Comment by Loreos7** [comment, marked as answer]
![Screenshot_20230406_103430.jpg](https://user-images.githubusercontent.com/86934170/230306895-0d48bd79-88cd-4edc-95b7-422c05a4c50f.jpg)

See https://osu.ppy.sh/wiki/en/Help_centre/Upgrading_to_lazer
